pyked/chemked.py

```diff
@@ -640,35 +640,109 @@ def __init__(self, properties):
         else:
             self.volume_history = None
 
-    def get_cantera_mole_fraction(self):
+    def get_cantera_composition_string(self, species_conversion=None):
         """Get the composition in a string format suitable for input to Cantera.
 
+        Returns a formatted string no matter the type of composition. As such, this method
+        is not recommended for end users; instead, prefer the `get_cantera_mole_fraction`
+        or `get_cantera_mass_fraction` methods.
+
+        Arguments:
+            species_conversion (`dict`, optional): Mapping of species identifier to a
+                species name. This argument should be supplied when the name of the
+                species in the ChemKED YAML file does not match the name of the same
+                species in a chemical kinetic mechanism. The species identifier (the key
+                of the mapping) can be the name, InChI, or SMILES provided in the ChemKED
+                file, while the value associated with a key should be the desired name in
+                the Cantera format output string.
+
         Returns:
             `str`: String in the ``SPEC: AMT, SPEC: AMT`` format
```

Comment:
This is old, but Cantera actually wants `SPEC:AMT, SPEC:AMT, SPEC:AMT` (i.e., no spaces after the colons)

Reply:
Oh, I thought I fixed that...